### PR TITLE
feat: support limit edges of one vertex before computing

### PR DESCRIPTION
**Pull request by imbajin**
Better to consider limit edges in input phrase (Later)

### CODE REVIEW

File: computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java [hunk 1]
```diff
@@ -73,14 +77,16 @@ public Edges edges(ReusablePointer vidPointer) {
                 long startPosition = this.input.position();
                 this.idPointer.read(this.input);
                 int status = vidPointer.compareTo(this.idPointer);
-                if (status < 0) { // No edges
+                // No edges
```
Comment (javeme) [inline computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java:80]: move to line 83

File: computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java [hunk 2]
```diff
@@ -207,10 +213,20 @@ public Edge next() {
...
+    /**
+     * Read edges & attach it by input stream, also limit the edges here
+     * TODO: use one reused Edges instance to read batches for each vertex.
+     */
     private Edges readEdges(RandomAccessInput in) {
         try {
+            // Limit edges to read here (unlimited by default)
             int count = in.readFixedInt();
+            // update count when "-1 < limitNum < count"
+            if (this.edgeLimitNum > UNLIMITED_NUM &&
```
Comment (javeme) [inline computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java:225]: it's more readable: `this.edgeLimitNum != UNLIMITED_NUM`

File: computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java [hunk 3]
```diff
@@ -207,10 +213,20 @@ public Edge next() {
...
     }
 
-    // TODO: use one reused Edges instance to read batches for each vertex.
+    /**
+     * Read edges & attach it by input stream, also limit the edges here
+     * TODO: use one reused Edges instance to read batches for each vertex.
+     */
     private Edges readEdges(RandomAccessInput in) {
         try {
+            // Limit edges to read here (unlimited by default)
```
Comment (javeme) [inline computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java:222]: seems it's too later to limit the degree here

Comment (imbajin) [reply computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java:222]: add a todo for better way (later)

**Review by corgiboygsj — APPROVED** [review]

**Review by zxxxhonest — APPROVED** [review]

**Pull request merged**